component/src/test/resources/activemq.properties

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Wrong licence year

Reply:
Fixed in 6bf095af9d7a4c00220069badb50517cec553139